### ISSUE TITLE
Safari rmc-shell not working

### ISSUE THREAD

**Issue by patkub** [patkub/rmc1891-site#78]
# It's a caching issue!

https://github.com/patkub/rmc1891-site/blob/master/app/src/rmc-shell.html does not load in Safari
PR https://github.com/patkub/rmc1891-site/pull/79

Saucelabs: https://circleci.com/gh/patkub/rmc1891-site/2121
WCT results:
```
windows 10 microsoftedge 15 (142/0/2)
windows 8.1 IE 11 (12/0/0 error)
os x 10.11 safari 9 (144/0/0)
macos 10.12 safari 10 (44/0/14 error)
macos 10.12 safari 11 (144/0/0)
```

<details> 
  <summary>safari 10 errors</summary>
<pre>
<code>
wct -s 'windows 10/microsoftedge@15' -s 'windows 8.1/internet explorer@11' -s 'os x 10.11/safari@9' -s 'macos 10.12/safari@10' -s 'macos 10.12/safari@11'
Creating Sauce Connect tunnel
Sauce Connect log: /tmp/wct11808-714-b2h5jl.dzxkm1v2t9/sc.log
Sauce tunnel active: 33bb5732-5fc1-4165-88bf-323ecfa1a4df
macos 10.12 safari 10    Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=3
os x 10.11 safari 9      Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=2
windows 8.1 IE 11        Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=1
macos 10.12 safari 11    Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=4
windows 10 microsoftedge 15 Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=0
windows 8.1 IE 11        Tests failed: Error thrown outside of test function: 'done' is undefined (http://localhost:8081/components/rmc1891/test/views/view-about.html?dom=shadow:57)
macos 10.12 safari 10    ✖ test/views/view-404.html » view-404 tests » Location /404

  null is not an object (evaluating 'view404.shadowRoot.querySelector')
    <unknown> at   <unknown> at view-404.html:44:0

macos 10.12 safari 10    ✖ test/views/view-404.html » view-404 tests » Go back home

  null is not an object (evaluating 'view404.shadowRoot.querySelector')
    <unknown> at   <unknown> at view-404.html:49:0

macos 10.12 safari 10    ✖ test/views/view-404.html » view-404 tests » Location /404

  null is not an object (evaluating 'view404.shadowRoot.querySelector')
    <unknown> at   <unknown> at view-404.html:44:0

macos 10.12 safari 10    ✖ test/views/view-404.html » view-404 tests » Go back home

  null is not an object (evaluating 'view404.shadowRoot.querySelector')
    <unknown> at   <unknown> at view-404.html:49:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » button should navigate to link

  null is not an object (evaluating 'rmcJumbotron.shadowRoot.querySelector')
    <unknown> at   <unknown> at rmc-jumbotron.html:49:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "text" property

  expected undefined to equal 'demo text'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:36:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "button" property

  expected undefined to equal 'demo button'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:40:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "link" property

  expected undefined to equal '/demo/path-example/'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:44:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » button should navigate to link

  null is not an object (evaluating 'rmcJumbotron.shadowRoot.querySelector')
    <unknown> at   <unknown> at rmc-jumbotron.html:49:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "text" property

  expected undefined to equal 'demo text'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:36:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "button" property

  expected undefined to equal 'demo button'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:40:0

macos 10.12 safari 10    ✖ test/elements/rmc-jumbotron.html » rmc-jumbotron tests » defines properties » defines the "link" property

  expected undefined to equal '/demo/path-example/'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-jumbotron.html:44:0

macos 10.12 safari 10    Tests failed: Error thrown outside of test function: null is not an object (evaluating 'rmcFooter.shadowRoot.querySelector')
  <unknown> at rmc-footer.html:29
macos 10.12 safari 10    ✖ test/elements/rmc-admin-tabs.html » rmc-admin-tabs tests » defines properties » defines the "active" property

  expected undefined to equal 'home'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-admin-tabs.html:32:0

macos 10.12 safari 10    ✖ test/elements/rmc-admin-tabs.html » rmc-admin-tabs tests » defines properties » defines the "active" property

  expected undefined to equal 'home'
    <unknown> at   AssertionError at /components/chai/chai.js:5553:0
    <unknown> at           assert at /components/chai/chai.js:206:0
    <unknown> at            equal at /components/chai/chai.js:2277:0
    <unknown> at        <unknown> at rmc-admin-tabs.html:32:0

macos 10.12 safari 11    Tests passed
windows 10 microsoftedge 15 ✖ test/elements/rmc-text-view.html » rmc-text-view tests » when editable » contians elements » it should have a what-you-see-is-what-you-get editor

  Object doesn't support property or method '_mutationObserverChanged'

os x 10.11 safari 9      Tests passed
windows 10 microsoftedge 15 ✖ test/elements/rmc-text-view.html » rmc-text-view tests » when editable » contians elements » it should have a what-you-see-is-what-you-get editor

  Object doesn't support property or method '_mutationObserverChanged'

</code>
</pre>
</details>

<details> 
  <summary>microsoftedge 15 errors</summary>
<pre>
<code>
windows 10 microsoftedge 15 Beginning tests via http://localhost:8081/components/rmc1891/generated-index.html?cli_browser_id=0
microsoftedge failed to maximize
windows 10 microsoftedge 15 ✖ test/elements/rmc-text-view.html » rmc-text-view tests » when editable » contians elements » it should have a what-you-see-is-what-you-get editor

  Object doesn't support property or method '_mutationObserverChanged'

windows 10 microsoftedge 15 ✖ test/elements/rmc-text-view.html » rmc-text-view tests » when editable » contians elements » it should have a what-you-see-is-what-you-get editor

  Object doesn't support property or method '_mutationObserverChanged'

windows 10 microsoftedge 15 Tests failed: 2 failed tests
Test run ended in failure: Error thrown outside of test function: 'done' is undefined (http://localhost:8081/components/rmc1891/test/views/view-about.html?dom=shadow:57), Error thrown outside of test function: null is not an object (evaluating 'rmcFooter.shadowRoot.querySelector')
  <unknown> at rmc-footer.html:29, 2 failed tests

windows 10 microsoftedge 15 (142/0/2) 
windows 8.1 IE 11 (12/0/0 error)      
os x 10.11 safari 9 (144/0/0)         
macos 10.12 safari 10 (44/0/14 error) 
macos 10.12 safari 11 (144/0/0)       

Error: Error thrown outside of test function: 'done' is undefined (http://localhost:8081/components/rmc1891/test/views/view-about.html?dom=shadow:57), Error thrown outside of test function: null is not an object (evaluating 'rmcFooter.shadowRoot.querySelector')
  <unknown> at rmc-footer.html:29, 2 failed tests

Exited with code 1

</code>
</pre>
</details>

**Comment by amritparmanand** [comment]
<details> 
  <summary>Safari Version 11.0.1 (13604.3.5)</summary>
<pre>
<code>
[Error] SyntaxError: Unexpected identifier 'Rogers'
	(anonymous function) (data:text/javascript;charset=utf-8,!fu…%0A:1:1362)
[Error] SyntaxError: Unexpected identifier 'Rogers' (common-worker-scope.js, line 2)
</code>
</pre>
</details>

**Comment by patkub** [comment]
@amritparmanand Thanks!

**Comment by patkub** [comment]
Closing as this should be fixed in 33205e3
